themes/bootstrap5/templates/cart/export.phtml

```diff
@@ -70,7 +70,7 @@
             $('form[name=exportForm]').removeAttr('target');
           } else {
             $('.export.btn').attr('data-lightbox-ignore', '1');
-            $('form[name=exportForm]').attr('target', this.selectedOptions[0].value + 'Main');
+            $('form[name=exportForm]').attr('target', this.selectedOptions[0].getAttribute('data-redirect'));
```

Comment:
Since we fetch the same attribute as part of the if statement, is it worth using a variable to avoid duplicate lookups? And is it maybe worth reducing jQuery usage here while we're revising it anyway? (Maybe that's overkill for the moment... but it feels like a missing part of the modernization).

Reply:
jQuery eliminated. :)